packages/swingset-theme-hashicorp/src/components/side-nav/category.tsx

```diff
@@ -0,0 +1,61 @@
+import { NavigationNode } from 'swingset/types'
+import { Link } from '../link'
+import { cx } from 'class-variance-authority'
+
+function Category({
+  title,
+  items,
+}: {
+  title: string
+  items: NavigationNode[]
+}) {
+  return (
+    <li className="ss-list-none">
+      <section>
+        <CategoryHeading>{title.toUpperCase()}</CategoryHeading>
+        <ComponentList items={items} />
+      </section>
+    </li>
+  )
+}
+//Swap this out for already existing heading
+function CategoryHeading({ children }: { children: string }) {
+  return <h4>{children.toUpperCase()}</h4>
+}
+
+function ComponentList({
```

Comment:
This should be renamed to `CategoryList` or `CategoryItems` for consistency with the other components